docs/scenarios.md

```diff
@@ -1,579 +1,466 @@
-base class parses an armory configuration file and calls a particular scenario's
-private `_evaluate` to perform all of the computation for a given threat-models
-robustness to attack. All `_evaluate` methods return a  dictionary of recorded metrics
-which are saved into the armory `output_dir` upon  completion.
+All scenarios inherit from the [Scenario](https://github.com/twosixlabs/armory/blob/master/armory/scenarios/scenario.py) class.
+This class parses an armory configuration file and calls its `evaluate` method to perform all of the computation for a given threat-models robustness to attack.
+All `evaluate` methods save a dictionary of recorded metrics which are saved into the armory `output_dir` upon  completion.
+Scenarios are implemented as subclasses of `Scenario`, and typically given their own file in the [Scenarios Directory](https://github.com/twosixlabs/armory/blob/master/armory/scenarios/).
+
+Of particular note is the [Poison](https://github.com/twosixlabs/armory/blob/master/armory/scenarios/poison.py) class, from which all poisoning scenarios are subclassed.
+More information on poisoning scenarios is documented [here](poisoning.md).
+
 
 ## Baseline Scenarios
 Currently the following Scenarios are available within the armory package.
+Some scenario files are tied to a specific attack, while others are customized for a given dataset.  Several are more general-purpose.
+Along with each scenario description, we provide a link to a page with baseline results for applicable datasets and attacks.
+More information about each referenced dataset can be found in the [datasets](datasets.md) document.
 
-### RESISC image classification (Updated June 2020)
 
+### Audio ASR (Updated June 2022)
 * **Description:**
-In this scenario, the system under evaluation is assumed to be a real-time overhead imagery scene classification
-system that a human operator is either passively monitoring or not monitoring at all.
+In this scenario, the system under evaluation is an automatic speech recognition system.
 * **Dataset:**
-The dataset is the [NWPU RESISC-45 dataset](http://www.escience.cn/people/JunweiHan/NWPU-RESISC45.html).
-It comprises 45 classes and 700 images for each class.  Images 1-500 of each class are in the training split,
-500-600 are in the validation split, and 600-700 are in the test split.
+  * Armory includes one dataset suited for ASR:
+    * [LibriSpeech dataset](http://www.openslr.org/12) (custom subset)
 * **Baseline Model:**
-To maximize time spent on defense research, a trained baseline model will be provided, but
-performers are not required to use it, if their defense requires a different architecture.
-The model is an ImageNet-pretrained DenseNet-121 that is fine-tuned on RESISC-45.
+Armory includes an audio model using the [DeepSpeech 2](https://arxiv.org/pdf/1512.02595v1.pdf) architecture with
+pretrained weights from either the AN4, LibriSpeech, or TEDLIUM datasets.  Custom weights
+may also be loaded by the model.
```

Comment:
There is also a HuBERT model available - see the eval6 scenario configs. I would add a deprecation warning to Deep Speech 2, and say that we'll be removing that in armory 0.17.

Reply:
done